spleeter_service/spleeter_service.py

```
@@ -0,0 +1,76 @@
+    def do_POST(self):
+        if self.path == '/process':
+            # Parse form data
+            content_type, params = cgi.parse_header(self.headers['content-type'])
+            if content_type != 'multipart/form-data':
+                self.send_error(400, "Content type must be multipart/form-data")
+                return
+
+            form = cgi.FieldStorage(
+                fp=self.rfile,
+                headers=self.headers,
+                environ={'REQUEST_METHOD': 'POST'}
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)